pkg/system/constants.go

```diff
@@ -8,6 +8,8 @@ const (
 	LEAccountSecretName  = "acorn-le-account"
 	DefaultUserNamespace = "acorn"
 	DNSSecretName        = "acorn-dns"
+	IngressName          = "acorn-dns-test-ingress"
```

Comment:
was thinking the variable name could be `DNSIngressName` and `DNSServiceName`, just to eliminate ambiguity.